extension.js

```diff
@@ -66,7 +72,10 @@ const BatteryConservationIndicator = GObject.registerClass(
       const status = Shell.get_file_contents_utf8_sync(sys_conservation);
       const active = (status.trim() == "1");
       this._indicator.visible = active;
-      this._item.label.text = _(`Turn Conservation Mode ${active ? 'Off' : 'On'}`);
+      if (active)
+      	this._item.label.text = _("Turn Conservation Mode Off")
+      else
+      	this._item.label.text = _("Turn Conservation Mode On");
```

Comment:
I changed this line because in languages like Spanish the verb turn on/off is a single word verb.